src/Library/demos/Grid/main.css

```
@@ -0,0 +1,27 @@
+#button00,
+#button01,
+#button02,
+#button10,
+#button11,
+#button12,
+#button20,
+#button21,
+#button22 {
```

in CSS, you should use a class to to make styling reusable between different elements

```suggestion
.cell {
```

You can apply the class to each button using the `styles ['cell']` syntax

And remove `name` on all `button`